### DISCUSSION TITLE
Welcome to supabase-client Discussions!

### DISCUSSION THREAD

**Discussion by CodeWithSushil** [CodeWithSushil/supabase-client#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by aguspuryanto** [comment]
when using $listTransactions = $client->getAllData('transactions'); only get 16 rows. 
how to get more or there any option for pagination

**Reply by CodeWithSushil** [reply]
Use official Supabase [client](https://github.com/supabase-community/supabase-php.git)

**Comment by CodeWithSushil** [comment]
Maybe using PostgreSQL api
